.travis.yml

```diff
@@ -4,42 +4,35 @@ php:
     - "5.6"
     - "7.2"
     - "7.3"
+    - "7.4"
 
 services:
     - mysql
 
 env:
     - WP_VERSION=latest WP_MULTISITE=0 #Current stable release
 
-# Test WP 5.1 and 5.2 and on PHP 5.6, 7.2 and 7.3 (w and w/o multisite enabled)
+# Test WP 5.2 and on PHP 5.6, 7.2, 7.3, and 7.4 (w/ and w/o multisite enabled)
```

Comment:
PR looks good, could tweak this `Test WP 5.2 and on` to be `Test WP 5.2 and WP Latest on` or get rid of the line entirely, the `matrix` section seems clear enough